components/match2/wikis/starcraft2/legacy/match_maps_legacy.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:MatchMaps/Legacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local String = require('Module:StringUtils')
+local Logic = require('Module:Logic')
+local PageVariableNamespace = require('Module:PageVariableNamespace')
+local Template = require('Module:Template')
+local Match = require('Module:Match')
+local MatchGroup = require('Module:MatchGroup')
+
+local globalVars = PageVariableNamespace()
+local matchlistVars = PageVariableNamespace('LegacyMatchlist')
+
+local MatchMapsLegacy = {}
+
+local _storageArgs
+
+local _OPPONENT_NUMBER = 2
```

Comment:
```suggestion
local _NUMBER_OPPONENTS = 2
```

Reply:
went with `_NUMBER_OF_OPPONENTS` it is even clearer imo^^